packages/experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -91,4 +110,15 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * A callback on whether this item can be removed
    */
   canRemoveItem?: (item: T) => boolean;
+
+  /** Map of callback functions related to row drag and drop functionality. */
+  dragDropEvents?: IDragDropEvents;
+
+  /**
+   * Helper for the drag and drop
+   */
+  dragDropHelper?: IDragDropHelper;
+
+  /** Event names and corresponding callbacks that will be registered to rendered selected items. */
+  selectedItemEventMap?: { eventName: string; callback: (context: IDragDropContext, event?: any) => void }[];
```

Comment:
i dont see this being used anywhere? what is the intent of this?

Reply:
It isn't. Copy pasted from an example and didn't use this. Removed in the most recent update.